api/src/main/java/org/openmrs/module/ModuleFileParser.java

```diff
@@ -518,21 +518,25 @@ private GlobalProperty extractGlobalProperty(Element element) {
 		String description = removeTabsAndTrim(getElementTrimmed(element, "description"));
 		String datatypeClassname = getElementTrimmed(element, "datatypeClassname");
 		String datatypeConfig = getElementTrimmed(element, "datatypeConfig");
+		String viewPrivilege = removeTabsAndTrim(getElementTrimmed(element, "viewPrivilege"));
+		String editPrivilege = removeTabsAndTrim(getElementTrimmed(element, "editPrivilege"));
+		String deletePrivilege = removeTabsAndTrim(getElementTrimmed(element, "deletePrivilege"));
 		
 		log.debug("property: {}, defaultValue: {}", property, defaultValue);
 		log.debug("description: {}, datatypeClassname: {}", description, datatypeClassname);
-		log.debug("datatypeConfig: {}", datatypeConfig);
+		log.debug("datatypeConfig: {}, viewPrivilege: {}", datatypeConfig, viewPrivilege);
```

Comment:
Shouldn't viewPrivilege be with its cousins (edit and delete privilege) instead of being with datatypeConfig? 😊

Reply:
Right, thought it would be in unison with the rest, but I changed it and put it with its cousins🙂